packages/web/src/components/notification/NotificationPanel.tsx

```diff
@@ -124,6 +118,10 @@ export const NotificationPanel = ({ anchorRef }: NotificationPanelProps) => {
     }
   }, [panelIsOpen, unviewedNotificationCount, dispatch])
 
+  const userHasNoNotifications =
```

Comment:
Can we invert this? Seems like it would be clearer as `userHasNotifications`

Reply:
totes